includes/integrations/class-member-chi-membership-plugin-integrations.php

```diff
@@ -28,16 +28,28 @@ public function define_hooks() {
 	 *
 	 */
 	public function load_integrations() {
-		require_once dirname(__FILE__) . '/class-member-chi-membership-plugin-integration.php';
+
+		require_once dirname( __FILE__ ) . '/class-member-chi-membership-plugin-integration.php';
 		if ( class_exists( 'RCP_Levels' ) ) {
-			require_once dirname(__FILE__) . '/restrict-content-pro/class-member-chi-restrict-content-pro-integration.php';
+			require_once dirname( __FILE__ ) . '/restrict-content-pro/class-member-chi-restrict-content-pro-integration.php';
 			new Member_Chi_Restrict_Content_Pro_Integration();
 		}
 
 		if ( class_exists( 'WPComplete' ) ) {
-			require_once dirname(__FILE__) . '/wp-complete/class-member-chi-wp-complete-integration.php';
+			require_once dirname( __FILE__ ) . '/wp-complete/class-member-chi-wp-complete-integration.php';
 			new Member_Chi_Wp_Complete_Integration();
 		}
+
+		if ( is_woocommerce_active() && is_plugin_active( 'woocommerce-memberships/woocommerce-memberships.php' ) ) {
```

Comment:
yeah, skyverge has some framework, which is also why you can't just check class_exists on WC_Memberships. This is their note from the function - 

>>>	/**
	 * Is the WooCommerce plugin installed and active? This method is handy for
	 * frameworked plugins that are listed on wordpress.org and thus don't have
	 * access to the Woo Helper functions bundled with WooThemes-listed plugins.
	 *
	 * Notice: For now you can't rely on this method being available, since the
	 * bootstrap class is the only piece of the framework which is loaded
	 * simply according to the lexical order of plugin directories. Therefore
	 * to use, you should first check that this method exists, or if you really
	 * need to check for WooCommerce being active, define your own method.
	 *
	 * @since 4.0.0
	 * @return boolean true if the WooCommerce plugin is installed and active
	 */